extensions/amp-live-list/0.1/live-list-manager.js

```diff
@@ -131,6 +139,12 @@ export class LiveListManager {
       url = addParamToUrl(url, 'amp_latest_update_time',
           String(this.latestUpdateTime_));
     }
+
+    if (this.isTransformed_) {
+      const urlService = Services.urlForDoc(this.ampdoc.getBody());
+      url = urlService.getCdnUrlOnCdnOrigin(this.location_, url);
```

Comment:
I think `this.location_` and `url` are always supposed to be the same (minus the additional params).